src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,41 @@
 import React from 'react';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import { Post } from '../../types/Posts';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
-);
+type Props = {
+  post: Post;
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => {
+  const {
+    title,
+    user,
+    body,
+    comments,
+  } = post;
+
+  return (
+    <div className="PostInfo">
+      <div className="PostInfo__header">
+        <h3 className="PostInfo__title">{title}</h3>
+
+        {user && (
+
+          <p>
+            {' Posted by  '}
+
+            {user && <UserInfo user={user} />}
```

Comment:
You already check it
```suggestion
          <p>
            {' Posted by  '}

            <UserInfo user={user} />
```